### ISSUE TITLE
If wrong template option is passed, output error is not clear

### ISSUE THREAD

**Issue by tjiagoM** [EpistasisLab/tpot#898]
I've updated TPOT to version '0.10.2', and if I run the following code:

```python
tpot = TPOTClassifier(generations=200, population_size=200, scoring='precision',
                          template='wrong template')
tpot.fit(X_train, y_train)
```

I receive a RuntimeError:
`RuntimeError: There was an error in the TPOT optimization process. This could be because the data was not formatted properly, or because data for a regression problem was provided to the TPOTClassifier object. Please make sure you passed the data to TPOT correctly.`

If I comment out the template option, TPOTClassifier starts running fine.

So I just want to suggest to improve this error output, as it doesn't say the true error (wrong template). If you wonder how the hell I came up to this error, it's because I was using a previous version of TPOT where "RandomTree" was a valid template, and after updating I was not understanding why the same code was giving me this error

**Comment by weixuanfu** [comment]
The default template is `None` since 0.10.2. We will refine error message in this case.

**Comment by weixuanfu** [comment]
The issue has been fixed in a new version of TPOT (v0.11.0). Please feel free to reopen it if you have any questions or suggestions.
